test/test.js

```diff
@@ -464,6 +464,14 @@ describe('Page', function() {
     });
   });
 
+  describe('Page.setDefaultNavigationTimeout', function() {
+    it('should work', async({page, server}) => {
```

Comment:
We generally try hard not to reach out to private fields.
Let's drop this test – i think the other one you add is sufficient.